frontend/src/lib/components/Animation/Animation.scss

```diff
@@ -0,0 +1,19 @@
+.Animation {
+    width: 100%;
+    max-width: 300px;
+    aspect-ratio: 1/1;
```

Comment:
This might need to change if we have other animations. Keeping it now as we don't have the dimensions before an animation loads... and a square is a square 🤔